Packs/Zoom/Scripts/ZoomAsk/README.md

```diff
@@ -59,6 +59,19 @@ The automation can utilize the interactive capabilities of Zoom to send a form w
 This requires the external endpoint for interactive responses to be available for connection. (See the Zoom integration documentation for more information).
 You can also utilize a dropdown list instead, by specifying the `responseType` argument.
 
+In order to use ZoomAsk via playbooks:
+    1. Add the ZoomAsk script to a playbook as a task.
+    2. In the 'Message' argument, specify the message to be sent.
+    3. Configure the response options by filling out the 'Option1' and 'Option2' arguments (default values are 'Yes' and 'No').
+    4. Either a user or a channel_id or channel_name must be specified.
+    5. All other inputs are optional.
+    6. At some point at the playbook, after running ZoomAsk, add a mutual conditional task which holds up the playbook execution until the response is received form zoom.
+       1. The condition names should much the response options you passed in to the ZoomAsk.
+       2. In order to tie the conditional task back to ZoomAsk:
+          - Add a tag to the conditional task.
+          - In the ZoomAsk task, include the same tag value under the "task" argument.
```

Comment:
```suggestion
          - In the `ZoomAsk` task, pass in the same tag value to the `task` argument.
```

Reply:
I would change the order here: I'd move this step up to right after step 4 when you are listing all the arguments that must be passed in to `ZoomAsk`.